app/web/features/profile/edit/EditProfile.tsx

```diff
@@ -134,13 +143,26 @@ export default function EditProfileForm() {
           },
         }
       );
+
+      if (showIncompleteProfileDialog) {
+        setShowIncompleteProfileDialog(false);
+      }
     },
     // All field validation errors should scroll to their respective field
     // Except the avatar, so this scrolls to top on avatar validation error
     (errors) =>
       errors.avatarKey && window.scroll({ top: 0, behavior: "smooth" })
   );
 
+  const handleSubmitButtonClick = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (aboutMeField?.length < 150) {
```

Comment:
Maybe we can move 150 into a constant. (It's something we've changed before too.)